templates/ingress.yaml

```diff
@@ -259,8 +259,8 @@ spec:
                 - ALL
               add:
                 - NET_BIND_SERVICE
-            # www-data -> 33
-            runAsUser: 33
+            # www-data -> 101
+            runAsUser: 101
```

Comment:
we don't store any files right? this won't affect any sort of filesystem permission things?

Reply:
they're all via configmaps it seems, didn't notice anything break in my testing

Reply:
no volumes, and i verified that the user is correct by pulling the image manually

Reply:
cool, sounds good, thanks for checking :+1: